zppy/mpas_analysis.py

```diff
@@ -21,6 +21,8 @@ def mpas_analysis(config, scriptDir):
     if len(tasks) == 0:
         return
 
+    campaign = config["default"]["campaign"]
```

Comment:
The changes in this file are unnecessary. I confirmed by running the integration tests with these changes removed. I also checked that `extra_config="--polar_regions"` for the cryosphere case.

Reply:
That's correct. For example, in `zppy/templates/mpas_analysis.bash`:
-  the variable ``{{ scriptDir }}`` comes from `c["scriptDir"] = scriptDir` in `zppy/mpas_analysis.py` (and in fact is not defined in `zppy/templates/default.ini` at all).
- the variable ``{{ debug }}``, on the other hand, is not defined in `main/zppy/mpas_analysis.py`` but rather only in ``zppy/templates/default.ini``. (This is the same case as `{{ campaign }}`).

Also worth noting is that the variables doesn't even have to be specified anywhere other than the bash file. For example, in `zppy/templates/e3sm_diags.bash`, ``{{ ref_start_yr }}`` isn't defined in `zppy/e3sm_diags.py` or in `zppy/templates/default.ini`. However, the user should specify that value in their config file if they are running the ENSO or QBO sets.

Reply:
Thanks.  I'll take these unnecessary parts out as soon as I get some feedback on the `e3sm_diags` set from @wlin7.